src/proxy.ts

```diff
@@ -200,7 +200,15 @@ export class ProxyBuilder {
       name: containerName,
       AttachStdout: true,
       AttachStderr: true,
-      Env: [`JOB_ID=${jobId}`],
+      Env: [
+        `http_proxy=${process.env.http_proxy}`,
```

Comment:
Since the proxy container only runs a single thing (vs the updater which can run code from multiple ecosystems) I think we could get away with only setting the lowercase versions here (see [here](https://about.gitlab.com/blog/2021/01/27/we-need-to-talk-no-proxy/) for support). However, we might want allow the user to specify it either way (`http_proxy=${process.env.http_proxy || process.env.HTTP_PROXY}`)? The [guidance](https://docs.github.com/en/actions/hosting-your-own-runners/using-a-proxy-server-with-self-hosted-runners) for actions runners is to use lowercase so we could stick to that too.

Reply:
👍 Great suggestion! Thanks for providing a link to the guidance 😄